### PR TITLE
Add configurable API server URL

### PR DESCRIPTION
**Pull request by gadhagod**
Addresses #18.
Example usage:

``` python
from os import getenv
import schoolopy
import yaml

sc = schoolopy.Schoology(schoolopy.Auth(getenv('SCHOOLOGY_KEY'), getenv('SCHOOLOGY_SECRET')), api_server='http://127.0.0.1:5000/v1')
print(sc.get_me())
```
**Note**: You probably want to update the README before merging if you are going to merge.

### CODE REVIEW

File: schoolopy/main.py [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
 
 class Schoology:
-    _ROOT = 'https://api.schoology.com/v1/'
     key = ''
     secret = ''
     limit = 20
     start = 0
 
-    def __init__(self, schoology_auth):
+    def __init__(self, schoology_auth, api_server='https://api.schoology.com/v1'):
```
Comment (ErikBoesen) [inline schoolopy/main.py:19]: I would prefer the variable be called `host` or `api_host`

File: schoolopy/main.py [hunk 2]
```diff
@@ -11,18 +11,18 @@
...
         if not schoology_auth.authorized:
             raise AuthorizationError('Auth instance not authorized. Run authorize() after requesting authorization.')
         self.key = schoology_auth.consumer_key
         self.secret = schoology_auth.consumer_secret
         self.schoology_auth = schoology_auth
+        self.api_server = api_server + '/'
```
Comment (ErikBoesen) [inline schoolopy/main.py:25]: I would prefer to have the / included in the parameter.

Comment (ErikBoesen) [reply schoolopy/main.py:25]: @AndrewLester wdyt?

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

File: schoolopy/main.py (hunk 2 above)
Comment (gadhagod) [reply schoolopy/main.py:25]: So should I get rid of ` + '/'`?

File: schoolopy/main.py (hunk 1 above)
Comment (gadhagod) [reply schoolopy/main.py]: Fixed in https://github.com/gadhagod/schoolopy/commit/46940ca43f68dff35f7de047dea6735a0ae55309.

File: schoolopy/main.py (hunk 2 above)
Comment (gadhagod) [reply schoolopy/main.py]: Fixed in https://github.com/gadhagod/schoolopy/commit/46940ca43f68dff35f7de047dea6735a0ae55309.

**Comment by gadhagod** [comment]
Requested changed resolved in https://github.com/gadhagod/schoolopy/commit/46940ca43f68dff35f7de047dea6735a0ae55309.

**Review by ErikBoesen — APPROVED** [review]

**Pull request merged**